cli/cmdlineparser.cpp

```diff
@@ -972,11 +976,13 @@ void CmdLineParser::printHelp()
               "                         analysis is disabled by this flag.\n"
               "    --check-library      Show information messages when library files have\n"
               "                         incomplete info.\n"
-              "    --clang              Experimental: Use Clang parser instead of the builtin\n"
-              "                         Cppcheck parser. Cppcheck executes `clang`. The Clang\n"
-              "                         AST is imported and converted into Cppcheck data.\n"
-              "                         After that the normal Cppcheck analysis is used. You\n"
-              "                         must have `clang` in PATH.\n"
+              "    --clang=<path>       Experimental: Use Clang parser instead of the builtin Cppcheck\n"
+              "                         parser. Takes the executable as optional parameter and\n"
+			  "                         defaults to `clang`. Cppcheck will run the given Clang\n"
```

Comment:
can you please replace tabs with spaces here the indentation looks wrong.

Reply:
Yeah - I has that right with the commit but I messed it up while resolving the merge conflict....

Reply:
Done